dev/ci/ci-common.sh

```diff
@@ -42,12 +42,14 @@ ls "$COQBIN"
 # Where we download and build external developments
 CI_BUILD_DIR="$PWD/_build_ci"
 
+ls "$CI_BUILD_DIR" || true
+
+set +x
```

Comment:
Actually the whole point of this was to allow parsing of the log to see if an overlay was selected, but I guess that's overkill. While you are at it you could make the `untar` silent too?